renderer/components/editor/options/export-select.js

```diff
@@ -27,8 +27,8 @@ Select.propTypes = {
   options: PropTypes.arrayOf(OptionType).isRequired
 };
 
-const ExportSelect = ({options, label}) => {
-  const hasOptions = options && options.length !== 0;
+const ExportSelect = ({options = [], label}) => {
```

Comment:
This also doesn't make sense. The proptype says it's required, so a default argument would then never be used.

Reply:
For the export buttons where there are no options, it will be undefined, propTypes for ExportSelect says it's optional:

Reply:
Wait, no: ```js
ExportSelect.propTypes = {
  options: PropTypes.arrayOf(OptionType),
  label: PropTypes.string.isRequired
};
```

it's not required in ExportSelect?

Reply:
The actual <Select/> component that just overlays and acts as the native view when you click it however, requires the options array to be set, which is conditionally checked via: `{hasOptions && <Select options={options}/>}`

Reply:
Yeah, nvm. I was looking at the wrong proptypes...

Reply:
Haha same here when I wrote that comment, then I double checked and saw that it was in fact so 😂

Reply:
Any suggestions on how to make that read better? rename ExportSelect to something else or move the actual select component underneath maybe?